archaius-zookeeper/src/main/java/com/netflix/config/source/ZooKeeperConfigurationSource.java

```diff
@@ -223,6 +224,10 @@ synchronized void deleteZkProperty(String key) throws Exception {
     }
     
     public void close() {
-    	Closeables.closeQuietly(pathChildrenCache);
+        try {
+            Closeables.close(pathChildrenCache, true);
+        } catch (IOException exc) {
+            logger.warn("IOException should not have been thrown.", exc);
```

Comment:
If you want to maintain the same behavior as before you should probably log this as ERROR since guava is logging it as SEVERE.